src/mumble/AudioOutput.cpp

```diff
@@ -660,17 +660,30 @@ bool AudioOutput::mix(void *outbuff, unsigned int frameCount) {
 					}
 				}
 
+				bool noPosVolume =
+					(Global::get().s.fAudioMaxDistVolume == 0) && (len >= Global::get().s.fAudioMaxDistance);
+				float maxDistVolume = calcGain(1.0f, len);
```

Comment:
```suggestion
				const bool noPosVolume =
					(Global::get().s.fAudioMaxDistVolume == 0) && (len >= Global::get().s.fAudioMaxDistance);
				const float maxDistVolume = calcGain(1.0f, len);
```
Plus I am wondering whether `isAudible` would be a better name for `noPosVolume` (with inverted value and meaning)